app/views/reports/regions/_definition_tooltip.html.erb

```diff
@@ -5,9 +5,24 @@
     data-placement="right"
     data-trigger="hover click"
     data-html="true"
-    title="<% definitions.each do |name, description| %>
-             <p class='mb-4px'><strong><%= name %>:</strong> <%= description %></p>
-           <% end %>"
+    title="
```

Comment:
Instead of this file, we reuse the tooltip component ([app/components/dashboard/card/tooltip_component.rb](https://github.com/simpledotorg/simple-server/pull/5091/files#diff-b4b68e4f3957eacf833076027ff90931a6e4005d3c4b4f630b2871f25a598a9e))